dockstore-webservice/src/main/resources/migrations.1.16.0.xml

```diff
@@ -191,4 +191,11 @@
         <modifyDataType columnName="topicmanual" newDataType="varchar(250)" tableName="tool"/>
         <modifyDataType columnName="topicmanual" newDataType="varchar(250)" tableName="workflow"/>
     </changeSet>
+    <changeSet author="hyunnaye" id="fixConceptDOIs">
+        <sql dbms="postgresql">
+            UPDATE workflow
+            SET conceptdoi = REGEXP_REPLACE(conceptdoi, 'doi/', '', 'g')
+            WHERE archived=false AND conceptdoi IS NOT NULL;
```

Comment:
Why when `archived = false`? If there's a reason for it, you might want to add a comment.

Reply:
FWIW, the "frozenness" of archived entries is enforced via a trigger that makes an update fail when the `archived` flag is `true` both before and after the update.  Essentially, the trigger causes updates to an archived entry to throw if they don't unarchive it.   So, if we did find we needed to make changes to archived entries, one possibility would be to disable the trigger, make the update, and then re-enable the trigger afterwards.  https://www.postgresql.org/docs/current/sql-altertable.html#SQL-ALTERTABLE-DESC-DISABLE-ENABLE-TRIGGER

Reply:
Disabled and reenabled the trigger in my new commit